### DISCUSSION TITLE
Tag and Image

### DISCUSSION THREAD

**Discussion by bern6** [mytourbook/mytourbook#1020] [General]
To linux users, picture added to tag is blurry, only Debian?
Otherwise great functionality.

**Comment by wolfgang-ch** [comment]
Can you attach here
* A MyTourbook screenshot with the blurry image
* The original image

**Reply by bern6** [reply]
Here are the different pictures (Linux 12, Windows 10, and Original)
![Debian 12](https://user-images.githubusercontent.com/103836286/224965986-eabfb951-5ee6-470d-b9d5-794c5e01a9c6.png)
![Windows 10](https://user-images.githubusercontent.com/103836286/224966204-d0d33738-0428-42e4-a5be-8f106c4991cc.png)
![Trek](https://user-images.githubusercontent.com/103836286/224966560-b206f11a-37ab-4a45-aa94-3f76f04251d8.jpg)

**Comment by wolfgang-ch** [comment]
> To linux users, picture added to tag is blurry, only Debian?

The images are not blurry, they are pixilated, blurry is the opposite.

I fixed it in https://github.com/mytourbook/mytourbook/commit/cd73c6e4672775273308f3b5027b8fc73f9e870b, these images are from my Ubuntu VM

---

Tag image size is 70 pixel, the upper image is the old

![image](https://user-images.githubusercontent.com/1283445/225019533-000d8166-d3b8-45ed-8a2f-b8f0952fee11.png)

---

Tag image size is 200 pixel (can be resized in the next version), the upper image is the old

![image](https://user-images.githubusercontent.com/1283445/225019717-9d4d30fb-cf92-46e1-b654-b2db639a14cf.png)
